epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -458,6 +402,44 @@ private void generateClassForModel(ClassToGenerateInfo info) throws IOException
     return methods;
   }
 
+  /**
+   * Generates default implementations of certain model methods if the model is abstract and doesn't
+   * implement them.
+   */
+  private Iterable<MethodSpec> generateDefaultMethodImplementations(ClassToGenerateInfo info)
```

Comment:
@elihart What are the pros and cons of creating empty implementation vs having the generated class be abstract?